docs/getting-started/quickstart.md

```diff
@@ -4,12 +4,24 @@ sidebar_position: 1
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import ThemedImage from '@theme/ThemedImage';
 
 # DBOS Quickstart
 
-This guide shows you how to deploy your first app to DBOS Cloud in less than 5 minutes. After that, it shows you how to develop DBOS apps locally.
+Let's create your first DBOS app! You start building an app by creating a special folder for it. From there, you can register with DBOS Cloud, provision a free Database Instance and deploy your app in a few minutes. You can also run DBOS apps on your computer.
```

Comment:
This still needs improvement. The paragraph is so dumbed down that it's confusing for someone who knows how to program. For example, the "special folder" isn't special in any way--it's just a regular folder with code in it. I still prefer my version:  "If you build an app with DBOS, you can develop it on your computer, then deploy it in seconds to DBOS Cloud," but I know it can also be improved.

Reply:
Should be all set now